### ISSUE TITLE
unmappable character for encoding error

### ISSUE THREAD

**Issue by CaledoniaProject** [GraxCode/JByteMod-Beta#10]
mvn complains about the following error, can you fix them?

```
src/me/grax/jbytemod/ui/JAboutFrame.java:[46,26] error: unmappable character for encoding UTF-8
src/me/grax/jbytemod/ui/PageEndPanel.java:[39,24] error: unmappable character for encoding UTF-8
```

**Comment by CaledoniaProject** [comment]
Removing the question mark seems to solved the problem, but I got another one ...

```
src/me/grax/jbytemod/ui/lists/MyCodeList.java:[27,29] error: package org.objectweb.asm.tree does not exist
```

**Comment by GraxCode** [comment]
try again now

**Comment by CaledoniaProject** [comment]
New errors with JDK8:

```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/InstrUtils.java:[87,31] error: incompatible types: Object cannot be converted to LabelNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/InstrUtils.java:[279,31] error: incompatible types: Object cannot be converted to LabelNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/ClassTree.java:[109,27] error: incompatible types: Object cannot be converted to MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/JarArchive.java:[113,39] error: incompatible types: Object cannot be converted to MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/JarArchive.java:[114,20] error: cannot find symbol
  symbol:   variable owner
  location: variable mn of type MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/JarArchive.java:[117,18] error: cannot find symbol
  symbol:   variable preLoad
  location: variable cn of type ClassNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/MyCodeList.java:[295,31] error: for-each not applicable to expression type
  required: array or java.lang.Iterable
  found:    InsnList
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/MyCodeList.java:[318,26] error: incompatible types: Object cannot be converted to FieldNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/SearchList.java:[99,31] error: incompatible types: Object cannot be converted to MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/SearchList.java:[169,31] error: incompatible types: Object cannot be converted to MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/TCBList.java:[28,36] error: incompatible types: Object cannot be converted to TryCatchBlockNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/LVPList.java:[29,37] error: incompatible types: Object cannot be converted to LocalVariableNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/asm/ParentUtils.java:[18,26] error: incompatible types: Object cannot be converted to String
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/MethodUtils.java:[68,7] error: size has private access in InsnList
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/MethodUtils.java:[70,7] error: size has private access in InsnList
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/MethodUtils.java:[90,25] error: cannot find symbol
  symbol:   variable owner
  location: variable mn of type MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/lpk/util/JarUtils.java:[80,37] error: incompatible types: Object cannot be converted to MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/lpk/util/JarUtils.java:[81,18] error: cannot find symbol
  symbol:   variable owner
  location: variable mn of type MethodNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/lpk/util/JarUtils.java:[84,16] error: cannot find symbol
  symbol:   variable preLoad
  location: variable cn of type ClassNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/analysis/converter/Converter.java:[95,31] error: incompatible types: Object cannot be converted to LabelNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/analysis/converter/Converter.java:[109,31] error: incompatible types: Object cannot be converted to LabelNode
[ERROR] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/dialogue/EditDialogue.java:[168,13] error: cannot find symbol
  symbol:   method setOpcode(int)
  location: variable ain of type AbstractInsnNode
[INFO] 22 errors
```

**Comment by CaledoniaProject** [comment]
And a few warnings

```
[WARNING] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/TCBList.java:[68,41] [unchecked] unchecked call to add(E) as a member of the raw type List
  where E is a type-variable:
    E extends Object declared in interface List
[WARNING] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/ui/lists/LVPList.java:[69,41] [unchecked] unchecked call to add(E) as a member of the raw type List
  where E is a type-variable:
    E extends Object declared in interface List
[WARNING] /private/tmp/JByteMod-Beta/src/me/grax/jbytemod/utils/asm/FrameGen.java:[40,23] [unchecked] unchecked call to addAll(Collection<? extends E>) as a member of the raw type List
  where E is a type-variable:
    E extends Object declared in interface List
[WARNING] /private/tmp/JByteMod-Beta/src/org/jetbrains/java/decompiler/modules/decompiler/LabelHelper.java:[442,11] [unchecked] unchecked conversion
  required: HashSet<Statement>[]
  found:    HashSet[]
[WARNING] /private/tmp/JByteMod-Beta/src/me/lpk/util/ASMUtils.java:[109,32] [unchecked] unchecked call to add(E) as a member of the raw type List
  where E is a type-variable:
    E extends Object declared in interface List
```

**Comment by GraxCode** [comment]
it should be fixed now, try again

**Comment by CaledoniaProject** [comment]
The code is compilable by now, thanks!

**Comment by CaledoniaProject** [comment]
Another runtime error .. missing dependency

```
%> java -jar target/JByteMod-1.3.0.jar
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.NoClassDefFoundError: org/fife/ui/rsyntaxtextarea/RSyntaxTextArea
```

**Comment by SubOptimal** [comment]
this should be solved with PR #12
